js/app/controllers/calendarlistcontroller.js

```diff
@@ -104,7 +104,7 @@ app.controller('CalendarListController', ['$scope', '$rootScope', '$window', 'Ha
 						$scope.subscription.newSubscriptionLocked = false;
 					})
 					.catch(function() {
-						OC.Notification.showTemporary(t('calendar', 'Error saving WebCal-calendar'));
```

Comment:
Please change this back to `Error saving WebCal-calendar` (and `Error saving WebCal-calendar` only)